cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -304,10 +304,11 @@ The following commands are only executable against the `system` database:
 |Command                                                                                                                                                                            | Admin category | Description
 | <<administration-database-aliases-alter-alias, ALTER ALIAS ... [IF EXISTS] SET DATABASE TARGET ...>>                                                                              | Database alias | Modifies the target of a database alias.
 | <<access-control-alter-password, ALTER CURRENT USER SET PASSWORD FROM ... TO>>                                                                                     | User and role  | Change the password of the user that is currently logged in.
-| <<administration-databases-alter-database, ALTER DATABASE ... [IF EXISTS] SET ACCESS {READ ONLY \| READ WRITE}>>                                                                  | Database       | Modifies the database access mode.
+| <<administration-databases-alter-database-access, ALTER DATABASE ... [IF EXISTS] SET ACCESS {READ ONLY \| READ WRITE}>>                                                                  | Database       | Modifies the database access mode.
+| <<administration-databases-alter-database-topology, ALTER DATABASE ... [IF EXISTS] SET TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}]>>                                                                  | Database       | Modifies the database topology.
```

Comment:
can these not be combined in one command, something like:
```
ALTER DATABASE name SET
ACCESS READ ONLY
TOPOLOGY 5 PRIMARIES
```
and therefore share a line same as create?

Reply:
have not looked that much at either CIP nor implementation to know if this is allowed

Reply:
according to the CIP it should be allowed (with double `SET` at least)...
```
ALTER DATABASE name [IF EXISTS] [SET ACCESS {READ ONLY | READ WRITE}]
[SET TOPOLOGY 
{ n PRIMAR{Y|IES} [IN "[" tag[, ...] "]" [ACROSS "[" tag[, ...] "]" ]] }+
[n SECONDAR{Y|IES} [IN "[" tag[, ...] "]" [ACROSS "[" tag[, ...] "]" ]]]*
]

```

Reply:
Looked at the implementation, it seems to allow it as well

Reply:
Yes, that was the intent when we wrote it anyway